src/renderer/components/App/AppIntro/AppIntroScreen.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div class="flex flex-row w-full h-full">
+    <div class="AppIntroScreen__container__left flex-1 animated fadeIn rounded-lg">
```

Comment:
I think it should follow the same responsiveness as the other screens:

```suggestion
    <div class="AppIntroScreen__container__left flex-1 animated fadeIn rounded-lg hidden lg:block">
```

Reply:
Thanks, removing the left part makes it better for smaller screens